src/api/resources/sample_units/benthiclitmethod.py

```diff
@@ -1,18 +1,14 @@
 from django.db import transaction
-from django.db.models import Sum
```

Comment:
For all the sample_unit files, there are significant number of deleted imports. I assume that's because they are unused -- just checking...

Reply:
yes